packages/model-viewer/src/test/features/scene-graph/material-spec.ts

```diff
@@ -176,63 +174,63 @@ suite('scene-graph/material', () => {
 
     test('test alpha cutoff expect valid value as default', async () => {
       await loadModel(ALPHA_BLEND_MODE_TEST);
-      expect(element.model!.materials[2].getAlphaCutoff()).to.be.equal(0.25);
+      expect(element.model!.materials[2]!.getAlphaCutoff()).to.be.equal(0.25);
     });
 
     test('test alpha cutoff test setting and getting', async () => {
       await loadModel(ALPHA_BLEND_MODE_TEST);
-      element.model!.materials[2].setAlphaCutoff(0.5);
-      expect(element.model!.materials[2].getAlphaCutoff()).to.be.equal(0.5);
+      element.model!.materials[2]!.setAlphaCutoff(0.5);
+      expect(element.model!.materials[2]!.getAlphaCutoff()).to.be.equal(0.5);
     });
 
     test('test double sided expect default is false', async () => {
       await loadModel(HELMET_GLB_PATH);
-      expect(element.model!.materials[0].getDoubleSided()).to.be.equal(false);
+      expect(element.model!.materials[0]!.getDoubleSided()).to.be.equal(false);
     });
 
     test('test double sided expect default is true', async () => {
       await loadModel(ALPHA_BLEND_MODE_TEST);
-      expect(element.model!.materials[1].getDoubleSided()).to.be.equal(true);
+      expect(element.model!.materials[1]!.getDoubleSided()).to.be.equal(true);
     });
 
     test('test double sided setting and getting', async () => {
       await loadModel(ALPHA_BLEND_MODE_TEST);
-      expect(element.model!.materials[1].getDoubleSided()).to.be.equal(true);
+      expect(element.model!.materials[1]!.getDoubleSided()).to.be.equal(true);
 
-      element.model!.materials[1].setDoubleSided(false);
-      expect(element.model!.materials[1].getDoubleSided()).to.be.equal(false);
+      element.model!.materials[1]!.setDoubleSided(false);
+      expect(element.model!.materials[1]!.getDoubleSided()).to.be.equal(false);
 
-      element.model!.materials[1].setDoubleSided(true);
-      expect(element.model!.materials[1].getDoubleSided()).to.be.equal(true);
+      element.model!.materials[1]!.setDoubleSided(true);
```

Comment:
Is there some way we could make some kind of a getter, where if you asked for a `null` material it would automatically load it so it would have something to give you? Maybe it's a helper method. Would be nice to avoid all these `!`s. But this can also be part of a later PR if you'd rather.

Reply:
Yeah we can lazy load the missing material.